static/app/components/nav/utils.tsx

```diff
@@ -60,13 +60,11 @@ export function isNavItemActive(
    * Issue submenu is special cased because it is matched based on query params
    * rather than the pathname.
    */
-  if (
-    location.pathname.includes('/issues/') &&
-    to.includes('/issues/') &&
-    to.includes('query=')
-  ) {
+  if (location.pathname.includes('/issues/') && to.includes('/issues/')) {
     if (location.search) {
-      return hasMatchingQueryParam({to, label: item.label}, location);
+      return (
+        hasMatchingQueryParam({to, label: item.label}, location) || item.label === 'All'
+      );
     }
     return item.label === 'All';
```

Comment:
🤔 Maybe we could DRY this up a little:
```js
const matchingParam = hasMatchingQueryParam({ ... }, location);
const isDefaultAll item.label === 'All';

if (locations.search) {
  return matchingParam || isDefaultAll;
}
return isDefaultAll;
```

Or maybe we could utilize the `if (location.search)` check and look at matching query params (not sure that function signature though - if it's boolean, we could toss that check there and let the condition fall through to the final return.

Reply:
Great suggestion! Addressed in c264fa2c5aa